docs/running.rst

```diff
@@ -115,6 +122,16 @@ Options:
 \-\-log\-api\-return\-codes
    Disable the :ref:`common-subexpression-elimination` optimization
 
+.. warning::
+
+    If multiple Solidity source files define the same contract name, you will get a single
+    compiled contract file for this contract name. As a result, you will only have a single
+    contract with the duplicate name without knowing from which Solidity file it originated.
+
+    Solang will not give a warning about this problem. This is the same behavior as
+    Ethereum Solidity.
```

Comment:
```suggestion
    Solang will not give a warning about this problem.
```

It is not the same behaviour as `solc`: 

```bash
❯ echo "contract A { function a() public pure {} }" > /tmp/a.sol && cp /tmp/a.sol /tmp/b.sol
❯ solc --bin -o /tmp/out /tmp/a.sol /tmp/b.sol 2>/dev/null
❯ ls /tmp/out/
_tmp_a_sol_A.bin  _tmp_b_sol_A.bin
```

It looks like `solc` just mangles the path into the contract artifacts. We don't need to fix that in this PR, but should line up with `solc` eventually later on.